### ISSUE TITLE
Fixing the build

### ISSUE THREAD

**Issue by h3xstream** [find-sec-bugs/find-sec-bugs#180]
The build is currently failing .. It is under investigation..
https://travis-ci.org/find-sec-bugs/find-sec-bugs/builds/110002111

**Comment by h3xstream** [comment]
Hum.. everything is compiling on my machine. Maybe I'm doing something different..
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] Find Security Bugs root ............................ SUCCESS [  1.362 s]
[INFO] FindBugs Test Utility .............................. SUCCESS [  4.474 s]
[INFO] Find Security Bugs plugin dependencies ............. SUCCESS [  2.737 s]
[INFO] Find Security Bugs Plugin .......................... SUCCESS [01:13 min]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:21 min
[INFO] Finished at: 2016-02-17T19:07:41-05:00
[INFO] Final Memory: 62M/524M
[INFO] ------------------------------------------------------------------------
```

**Comment by formanek** [comment]
I think there is no problem in the new code (the build was failing with string change only too) and the old code is probably also fine (it never failed like this before), it looks like something changed in Travis and I hope we can fix it by modifying `.travis.yml` file. VM can crash because of insufficient memory, maybe setting memory by adding JVM arguments like `-Xmx` could help, [somebody used](https://stackoverflow.com/questions/23260057/the-forked-vm-terminated-without-saying-properly-goodbye-vm-crash-or-system-exi#link-post-33757854) `-Xmx1024m -XX:MaxPermSize=256m` to solve the issue with Travis.

**Comment by h3xstream** [comment]
Ok it should be fixed in 6e3e4a9861054b5d6e3b2194c9eaeb8208a252ac
https://travis-ci.org/find-sec-bugs/find-sec-bugs/builds/110455839

@formanek Thanks for the hint
